### PR TITLE
Create Instructions

### PR DESCRIPTION
**Pull request by mattdavis0351**
@hectorsector @lucie-docs Can you two please take a look at this README.md and improve the instructions for this exercise.

@hectorsector Second to that, currently `main` has everything it needs for manual testing.  I didn't provide automated testing for this one since we don't do anything super custom.  It doesn't make much sense to write tests for libraries that are already thoroughly tested.

We are just crafting payloads based on event results. 

Don't wait on me to merge this.  Once you two decide it's good to go then we should merge this in 😄

### CODE REVIEW

**Comment by hectorsector** [comment]
@mattdavis0351 @lucie-docs I pushed up an opinionated change in 1731b7b and 8b4027f, which requires the learner to figure out how to use the complete key/value pair in the exercise. I think this is part of the objective here, which is to use the secret appropriately (not just to create it).

**Comment by mattdavis0351** [comment]
@hectorsector ~I get where you are coming from, however without access to documentation about the local action that is running, the learner will **never know** what that the action expects the input to be named **your-secret**.~

~So from my point of view, we have to at least let them know that the input is named **your-secret** somewhere.~

>how to use the complete key/value pair in the exercise

~The usual way is by viewing the actions README.md that get's published on the marketplace~

I see the previous changes now 😛 🤦

File: README.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ This lab is an exercise to check your knowledge on scheduled events. It is autom
 Please complete the instructions below:
 
 1. Create your own copy of this repository by using the [Use this template](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template#creating-a-repository-from-a-template) button.
+2. Create a new, or use an existing Personal Access Token that has the `repo` scope enabled.
+3. Create a new secret in your copy of this repository that contains the value of the Personal Access Token you wish to use.
+4. Pass the secret to the local action `check-secrets-action` using the key `your-secret`. The action is used in a workflow titled `use-secrets.yml`.
```
Comment (mattdavis0351) [inline README.md]: @hectorsector I think specifying 'input' helps the learner to understand that their key belongs in the 'with:' block for the action.

```suggestion
4. Pass the secret to the local action `check-secrets-action` using the input key `your-secret`. The action is used in a workflow titled `use-secrets.yml`.
```

**Review by hectorsector — APPROVED** [review]
Let's merge this in so we can do some user testing! @lucie-docs feel free to open a new PR against these instructions when you're back ❤️

**Pull request merged**